docs/src/tutorials/tutorial_2.md

```diff
@@ -26,67 +24,86 @@ When the original model calls [`set_param!`](@ref), Mimi creates an external par
 ```julia
 update_param!(mymodel, :parametername, newvalues) # update values only 
 
-update_param!(mymodel, :parametername, newvalues, update_timesteps=true) # also update time keys
+update_param!(mymodel, :parametername, newvalues, update_timesteps=true) # also update time keys. Only necessary if the time dimension of the model has been changed.
 ```
 
-Also note that in the code above,`newvalues` must be the same size and type (or be able to convert to the type) of the old values stored in that parameter.
+Also note that in the code above, `newvalues` must be the same size and type (or be able to convert to the type) of the old values stored in that parameter.
 
 If you wish to alter connections within an existing model, [`disconnect_param!`](@ref) and [`connect_param!`](@ref) can be used in conjunction with each other to update the connections within the model, although this is more likely to be done as part of larger changes involving components themslves, as discussed in the next subsection.
 
 ## Parametric Modifications: DICE Example
 
-### Step 1. Download DICE
+### Step 1. Download MimiDICE2010
 
-The first step in this process is downloading the FUND model, which is now made easy with the Mimi registry.  Assuming you have already done the one-time run of the following to connect your julia installation with the central Mimi registry of Mimi models
+The first step in this process is downloading the DICE2010 model, which is now made easy with the Mimi registry. Assuming you have already done the one-time run of the following to connect your julia installation with the central Mimi registry of Mimi models:
 
 ```julia
 pkg> registry add https://github.com/mimiframework/MimiRegistry.git
 ```
 
-You simply need to add the DICE2010 model with 
-```
-add MimiDICE2010
+You simply need to add the MimiDICE2010 model in the Pkg REPL with:
+```julia
+pkg> add MimiDICE2010
 ```
-You have now successfully downloaded DICE to your local machine.
+You have now successfully downloaded MimiDICE2010 to your local machine.
 
 ### Step 2. Run DICE
 
-The next step is to run DICE.  If you wish to first get more aquainted with the model itself, take a look at the provided online documentation.  Now run DICE using the provided API for the package:
+The next step is to run DICE using the provided API for the package:
 
-```
+```jldoctest tutorial2; output = false, filter = r".*"s
 using MimiDICE2010
```

Comment:
It seems like all of the tutorial blocks now have `output = false`, does that mean they aren't being tested?